README.md

```diff
@@ -1 +1,54 @@
-# space-travelerers
\ No newline at end of file
+# JFF Space Travelers Hub
+## 📑 Project description
+
+The Space Travelers Hub project was built using ReactJS, SCSS, HTML, and tested through Jest.
+This is a React/Redux project for educational purposes only.
+
+## 🛠 Built with
+- ReactJS / Redux
+- HTML5
+- CSS3
+
+## 📺 Live Demo
+Live version: NOT YET
```

Comment:
- [ ] [OPTIONAL] Please deploy your project as soon as possible and put the deployment link in your README file. This will be very good for your project documentation.